advanced_alchemy/config/asyncio.py

```diff
@@ -63,3 +63,7 @@ class SQLAlchemyAsyncConfig(GenericSQLAlchemyConfig[AsyncEngine, AsyncSession, a
 
     The configuration options are documented in the Alembic documentation.
     """
+
+    def __post_init__(self) -> None:
+        if self.metadata:
+            self.alembic_config.target_metadata = self.metadata
```

Comment:
Make sure to call `super().__post_init__()`, but might be better just to do this once on the base class?

Reply:
Good call.  I'll add it here for now.